R/object-rc.R

```diff
@@ -24,7 +24,7 @@ rc_methods <- function(obj) {
   parent_methods <- unlist(lapply(parents, function(x) {
     methods::getRefClass(x)$methods()
   }))
-  method_names <- setdiff(ls(envir = obj@refMethods), parent_methods)
```

Comment:
How about just using `sort_c()` here? That makes it very clear that the results are sorted.

Reply:
Makes sense! Updated